Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -141,13 +141,32 @@ - (void)addEnabledCrumb
     [SentrySDK addBreadcrumb:crumb];
 }
 
-- (void)swizzleSendAction
++ (BOOL)avoidSender:(id)sender forTarget:(id)target action:(NSString *)action
 {
 #if SENTRY_HAS_UIKIT
+    if ([sender isKindOfClass:UITextField.self]) {
+        // This is required to avoid creating breadcrumbs for every key pressed in a text field.
+        // Textfield may invoke many types of event, in order to check if is a
+        // `UIControlEventEditingChanged` we need to compare the current action to all events
+        // attached to the control. This may cause a false negative if the developer is using the
+        // same action for different events.
+        UITextField *textField = sender;
+        NSArray<NSString *> *actions = [textField actionsForTarget:target
+                                                   forControlEvent:UIControlEventEditingChanged];
```

Comment:
`m`: Should we maybe also consider to ignore [UIControlEventValueChanged](https://developer.apple.com/documentation/uikit/uicontrolevents/uicontroleventvaluechanged?language=objc)?

Reply:
UITextField don't throw `UIControlEventValueChanged`